### PR TITLE
Draft of IMessageService and IMessageHistoryService

### LINKED ISSUE #12: create services for message, message history
Create Services for Models

### PR DESCRIPTION
**Pull request by IruomaJennifer**

### CODE REVIEW

File: CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CCSA_ChatApp.Infrastructure.Services
+{
+    public interface IMessageHistoryService
+    {
+        List<MessageDTO> FetchMessagesBySenderId(Guid senderId);
```
Comment (giddy11) [inline CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs]: IEnumerable for List... Buh it's fine to me though

Comment (Adexandria) [reply CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs]: Pls use Ienumerable

Comment (IruomaJennifer) [reply CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs]: I guess its only a problem for fluent nhibernate, I'm not sure though

Comment (IruomaJennifer) [reply CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs]: Okay I'll change it

**Review by giddy11 — APPROVED** [review]

**Review by giddy11 — APPROVED** [review]
thks

**Pull request merged**